week7/JungSangHyun/src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,72 @@
+import OrderBar from "../../components/OrderBar/OrderBar";
+
+const Home = () => {
+
+  interface Food{
+    name: string;
+    img: string;
+  }
+
+  const foods :Food[] = [
+    { name: "피자", img: "/pizza.svg" }, { name: "샐러드", img: "/salad.svg" }, { name: "햄버거", img: "/hamburger.svg" },
+    { name: "한식", img: "/koreanFood.svg" }, { name: "분식", img: "/schoolFood.svg" }, { name: "치킨", img: "/chicken.svg" },
+    { name: "초밥", img: "/sushi.svg" }, { name: "샌드위치", img: "/sandwich.svg" }, { name: "파스타", img: "/pasta.svg" },
+    { name: "디저트", img: "/donut.svg" }, { name: "커피", img: "/coffee.svg" }
+  ];
+
+  // function
+  const openMoreFoods = () => {
+
+  }
+
+  return (
+    <div className="mx-auto w-[390px] h-[844px] flex flex-col justify-between bg-white">
+      <section className="flex flex-col gap-[13px] pt-[25px]  pr-[68px] pb-[70px]">
+          <h3 className="text-[#191F28] font-pretendard text-[26px] font-bold leading-normal">
+            오늘은 무엇을 먹을까요?
+          </h3>
+        
+          <div className="flex gap-1">
+            <span className="text-[#333D4B] font-pretendard text-[17px] font-medium leading-normal">
+            한남중앙로 40길 (한남 빌리지)(으)로 배달
+            </span>
+          
+            <button className="cursor-pointer text-[#333D4B] font-pretendard text-[17px] font-medium leading-normal">
+                &gt; 
+            </button>
+          </div>
+          
+      </section>
+
+      <section className="pb-50">
+        <ul className="grid grid-cols-3 gap-[9px] justify-center">
+         
+          {foods.map((food) => (
+            <li
+              key={food.name}
+              className="flex w-[108px] pt-[12px] pb-[13px] flex-col  justify-center
+                        items-center gap-[4px] rounded-[8px] bg-[#FAFAFB] cursor-pointer hover:bg-gray-200"
+            >
+              <img src={food.img} alt={`${food.name} 이미지`} className="w-7 h-7" />
```

Comment:
tailwind 중에 size-<value>가 있는데 요거를 쓰면 width랑 height 값이 같을 때 한번에 지정 가능합니다.
```suggestion
<img src={food.img} alt={`${food.name} 이미지`} className="size-7" />
```